server/index.js

```diff
@@ -139,7 +142,7 @@ app.use(
                 logo()
               ),
               typeof component === 'function'
-                ? component(req.context)
+                ? component(req.context, req.states)
```

Comment:
See previous comment about `context`.

```suggestion
                ? component(req.context)
```